packages/core/tooling/index.ts

```diff
@@ -238,3 +239,11 @@ export function guessQuoteStyle(ast: TsEstree.Node): 'single' | 'double' | undef
 
 	return singleCount > doubleCount ? 'single' : 'double';
 }
+
+export function parseYaml(content: string): ReturnType<typeof yaml.parseDocument> {
+	return yaml.parseDocument(content);
+}
+
+export function serializeYaml(data: unknown): string {
```

Comment:
Not on the right computer to confirm this, but couldn't the param type here be the same as the return type of parse? `ReturnType<typeof yaml.parseDocument>`

Reply:
Yes, correct ✌️